CONTRIBUTING.md

```diff
@@ -0,0 +1,155 @@
+# Contributing to templates
+
+This page describes how to maintain and contribute to this templates repository.
+You'll learn about the structure of templates and how to document them.
+
+Broadly, use the regular [Data Management Development Workflow](https://developer.lsst.io/processes/workflow.html) when contributing to this repository.
+Because of the broad impact of these templates, a [Request for Comments (RFC)](https://developer.lsst.io/processes/decision_process.html) may be appropriate.
```

Comment:
I think it would be good at some point (maybe not on this PR) to outline some examples for when an RFC would/would not be needed.